patientsearch/src/js/components/TimeoutModal.js

```diff
@@ -94,9 +102,15 @@ export default function TimeoutModal() {
   };
 
   const handleClose = () => {
+    clearExpiredIntervalId();
     setOpen(false);
   };
 
+  const reLoad = () => {
+    handleClose();
+    location.reload();
+  }
```

Comment:
To force-request a new Access Token (when one is about to expire, but still valid), I added a new API endpoint to locally log someone out of flask_oidc (and return them to `/`): `/clear_session`

Instead of reloading the page, can you redirect to this new endpoint?

When a user is returned to `/` (by `/clear_session`), they should have a new Access Token, or be forced to log in again if their Keycloak session has expired

Reply:
thanks. fixed